ignite/distributed/utils.py

```diff
@@ -46,19 +45,13 @@
 _need_to_sync = True
 
 
-def _sync_model_wrapper(func):
```

Comment:
Here, I don't understand why the decoration leads to the overhead.

Reply:
It is a small overhead (~1e-6 on my pc) but of the same order as the execution of wrapped function (e.g. `dist.get_world_size()`).